frontend/pages/beta/model/[modelId]/schema.tsx

```diff
@@ -90,6 +90,7 @@ function SchemaButton({ modelId, schema }: SchemaButtonProps) {
   async function createModelUsingSchema(newSchema: SchemaInterface) {
     if (currentUser && model) {
       await postFromSchema(model.id, newSchema.id)
+      await mutateModel()
```

Comment:
You don't need to await this mutate as we aren't expecting a response, it should happen in the background.